memstore/list/live_vector_party.go

```diff
@@ -55,10 +52,22 @@ func (vp *LiveVectorParty) GetBytes() int64 {
 	return bytes
 }
 
+// GetTotalBytes return the bytes this vp occpuies including memory pool
+func (vp *LiveVectorParty) GetTotalBytes() int64 {
+	bytes := vp.GetBytes()
+
+	vp.RLock()
```

Comment:
could new data be inserted between lock released from line 57 and new lock grabbed here..?

Reply:
updated, this function is usually used in testing